lib/ansible/modules/web_infrastructure/ansible_tower/tower_job_launch.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python
+#coding: utf-8 -*-
+
+# (c) 2017, Wayne Witzel III <wayne@riotousliving.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: tower_job_launch
+version_added: "2.3"
+short_description: Launch an Ansible Job.
+description:
+    - Launch an Ansible Tower jobs. See
+      U(https://www.ansible.com/tower) for an overview.
+options:
+    job_template:
+      description:
+        - Name of the job_template to use.
+      required: True
+    job_explanation:
+      description:
+        - Job explanation field.
+      required: False
+      default: null
+    job_type:
+      description:
+        - Job_type to use for the job, only used if prompt for job_type is set.
+      required: False
+      choices: ["run", "check", "scan"]
+      default: null
+    inventory:
+      description:
+        - Inventory to use for the job, only used if prompt for inventory is set.
+      required: False
+      default: null
+    credential:
+      description:
+        - Credential to use for job, only used if prompt for credential is set.
+      required: False
+      default: null
+    extra_vars:
+      description:
+        - Extra_vars to use for the job_template. Use '@' for a file.
+      required: False
+      default: null
+    limit:
+      description:
+        - Limit to use for the job_template.
+      required: False
+      default: null
+    tags:
+      description:
+        - Specific tags to use for from playbook.
```

Comment:
We expect this to be a comma separated list, `foo, bar, baz` but internally we expect that list to be a string. So I guess we could set it to `type=list` and I can just `.join` it before I pass it in to our Tower library that makes the call.